nx/lib/nx/serving.ex

```diff
@@ -873,6 +879,11 @@ defmodule Nx.Serving do
       The number of partitions will be determined according to your compiler
       and for which host it is compiling. See the module docs for more information
 
+    * `:weight` - weight used for load balancing when running a distributed serving
+      Defaults to `1`. If it is set to a higher number `w`, the serving process
+      will receive, on average, `w` times the number of requests compared to a
+      process that uses `weight: 1`.
```

Comment:
We should document this is multiplied by the number of partitions?

Reply:
Added a sentence in the docstring.